src/app/modules/main/communities/view.nim

```diff
@@ -495,6 +495,9 @@ QtObject:
   proc isCommunityRequestPending*(self: View, communityId: string): bool {.slot.} =
     self.delegate.isCommunityRequestPending(communityId)
 
+  proc removePrivateKey*(self: View, communityKey: string) {.slot.} =
```

Comment:
```suggestion
  proc removePrivateKey*(self: View, communityId: string) {.slot.} =
```

Reply:
Shouldn't be `communityId`? Same for all proc

Reply:
Fixed!